core/object/message_queue.cpp

```diff
@@ -34,6 +34,7 @@
 #include "core/core_string_names.h"
 #include "core/object/class_db.h"
 #include "core/object/script_language.h"
+#include <cstdio>
```

Comment:
We separate Godot headers from system headers with an empty line:
```suggestion
#include "core/object/script_language.h"

#include <cstdio>
```